lib/openstudio-standards/prototypes/common/objects/Prototype.Fan.rb

```diff
@@ -174,6 +174,7 @@ def create_fan_by_name(model,
                                   motor_in_airstream_fraction: motor_in_airstream_fraction,
                                   end_use_subcategory: end_use_subcategory)
     elsif fan_json['type'] == 'VariableVolume'
+      fan_power_coefficient_1, fan_power_coefficient_2, fan_power_coefficient_3, fan_power_coefficient_4, fan_power_coefficient_5 = lookup_fan_curve_coefficients_from_json(fan_json['fan_curve']) if fan_json['fan_curve']
```

Comment:
where are these coefficients applied?

Reply:
This retrieves the fan coeficients from the curves json file, if a curve name is specified in the fan json file. The coefficients are then applied in create_fan_variable_volume.